CMSgov/design-system
# [WNMGDS-1888] Fix inverse medicare drawer toggle

## Summary

https://jira.cms.gov/browse/WNMGDS-1888

- Fix bad reference to obsolete button class name in the medicare overrides

## How to test

I checked out a release prior to our buttons breaking change and added this story so I could verify what the inverse toggle looked like. It should look like this:

<img width="161" alt="Inverse toggle" src="https://user-images.githubusercontent.com/7595652/187014644-b1e46cf7-73dd-4bdb-b116-62e048dc93d5.png">

(hover)
<img width="157" alt="Inverse toggle hover" src="https://user-images.githubusercontent.com/7595652/187014642-51548c0d-5138-4a94-8b29-732e7df1163a.png">

### If this is a change to code:
